src/app/content/components/popUp/ChapterFilter.tsx

```diff
@@ -130,6 +130,7 @@ const ChapterFilter = (props: ChapterFilterProps) => {
               title={section.title}
               onChange={() => handleChange(section)}
               ariaLabel={getAriaLabel(section)}
+              dataAnalyticsLabel={`Filter PQ by ${splitTitleParts(section.title).join(' ')}`}
```

Comment:
or just create function like `prepareSectionTitle` and use it here and in `getAriaLabel`

Reply:
> connect these functions somehow so we are sure that if one changes then we change another one

What exactly do you mean? As I understand there is no other option than to _hardcode_ the `data-analytics-label`.

Reply:
I mean that you are reusing `splitTitleParts(section.title).join(' ')` from `getAriaLabel` function
Instead of doing this you could have `getAriaLabel(section)` and `getAnalyticsLabel(section)` and both of them would use `prepareSectionTitle(section) => splitTitleParts(section.title).join(' ')`
However I have second thoughts on this because we don't want to change analytics labels when we change aria labels so it might be better to leave it as it is now